janus/lib/client/jsx/janus.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import Cookies from 'js-cookie';
+
+import createStore from './store';
+import JanusUI from './janus-ui';
+
+function Janus() {
+  this.store = createStore();
```

Comment:
Hmm, should this be a const or useState instead of a `this.store` reference?  The `this` here will become the window, so we're putting the store in a global.  It works, but it may be safer to keep it more local.